jib-core/src/main/java/com/google/cloud/tools/jib/filesystem/XdgDirectories.java

```diff
@@ -132,7 +133,7 @@ private static Path getOsSpecificDirectory(
       }
       Path localAppData = Paths.get(localAppDataEnv);
       if (!Files.exists(localAppData)) {
-        LOGGER.warning(localAppData + " does not exist");
+        LOGGER.log(Level.WARNING, "{} does not exist", localAppData);
```

Comment:
Are you sure `{}` works? The [Javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Logger.html) of JUL (aka `java.util.logging`) `Logger` says

> Typically, formatters use java.text.MessageFormat style formatting to format parameters, so for example a format string `"{0} {1}"` would format two parameters as strings.

Reading the `MessageFormat` [Javadoc](https://docs.oracle.com/javase/8/docs/api/java/text/MessageFormat.html), I think it should contain a number?

Also, doesn't `LOGGER.warning()` have an override that accepts message arguments?

Reply:
> Are you sure {} works?

Thanks for pointing out! I was mistaken with log4j formats. And you are right,`{0}` is the way to go.
> Also, doesn't LOGGER.warning() have an override that accepts message arguments?

Ah, perfect. I think this is what I needed. `warning(Supplier<String> msgSupplier)` this accepts a supplier and will [only to be constructed if the logging level is such](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Logger.html#warning-java.util.function.Supplier-).